src/Pact/Eval.hs

```diff
@@ -755,7 +756,7 @@ fullyQualifyDefs info mdef defs = do
           && mn == _mnName (_mName mdef)
           && isNsMatch -> resolveBareName memo (BareName fn i)
           where
-            isNsMatch = fromMaybe True (liftA2 (==) modNs mNs)
+            isNsMatch = fromMaybe True (Control.Applicative.liftA2 (==) modNs mNs)
```

Comment:
I think we can just do `(==) <$> modNs <*> mNs` instead and avoid `liftA2` entirely.

Reply:
i'd rather just expand in terms of `<*>` than have an extra qualified import :)

Reply:
I completely agree, just needed confirmation :) I've just pushed a commit replacing `liftA2`.